src/Loader/AbstractWebpackLoader.php

```diff
@@ -192,6 +192,10 @@ protected function resolveLocation(string $fileName): int
         }
 
         if (stristr($fileName, '-block')) {
+            return Asset::BLOCK_EDITOR_ASSETS | Asset::BLOCK_ASSETS;
```

Comment:
`return Asset::BLOCK_EDITOR_ASSETS | Asset::BLOCK_ASSETS;` this probably is too much and just `Asset::BLOCK_ASSETS;` it's enough for both contexts.